github-linguist.gemspec

```diff
@@ -2,7 +2,7 @@ require File.expand_path('../lib/linguist/version', __FILE__)
 
 Gem::Specification.new do |s|
   s.name    = 'github-linguist'
-  s.version = Linguist::VERSION
+  s.version = ENV['GEM_VERSION'] || Linguist::VERSION
```

Comment:
Added this to make it easy to build test gems with version numbers that don't cause caching problems using:

```
GEM_VERSION=$(git describe --tags 2>/dev/null | sed 's/-/./g' | sed 's/v//') bundle exec rake build_gem
```

... which gives us cache-friendly names like: `github-linguist-5.3.3.50.g6186660e.gem`